lib/core/src/packStruct.cpp

```diff
@@ -2744,18 +2725,15 @@ unpackPointerItem( packItem_t &myPackedItem, const void *&inPtr,
         break;
     case PACK_DOUBLE_TYPE:
         if ( myDim == 0 ) {
-            outPtr = addPointerToPackedOut( unpackedOutput,
-                                            numElement * elementSz, NULL );
-            status = unpackDoubleToOutPtr( inPtr, outPtr, numElement,
-                                           myPackedItem.name, irodsProt );
+            outPtr = addPointerToPackedOut( unpackedOutput, numElement * elementSz, NULL );
+            unpackDoubleToOutPtr( inPtr, outPtr, numElement, myPackedItem.name, irodsProt );
```

Comment:
This line now mirrors the master branch. Is it a bug in master that the result is not captured in `status`? I'm not sure which implementation is the correct one, but capturing the result in `status` does feel more correct.

Reply:
so the comment says don't check status... because it could be negative... that seems... weird/incorrect?

Reply:
Yes, that is weird. I didn't see that before.

So 4-2-stable should be updated to mirror master? Both PRs passed CI even with this difference.

Reply:
It looks like the comment is saying to literally not check the captured value because it will be checked later. If that is the case, maybe the return value should be captured like before.

Reply:
I think I see what is happening. The assignment to `status` is wasted cpu cycles because it is never read after this point. Removing it does not affect the outcome of this function at all.

Reply:
Notice there are no loops which allow status to be checked later. On top of that, each case statement that cares about the result checks `status` immediately after it is updated.

Reply:
agreed.   carry on.